sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -38,6 +39,8 @@ export class BatchingReceiver extends MessageReceiver {
 
   private _finalActionHandler: (() => void) | undefined = undefined;
 
+  private _drainTimeoutInMs: number = receiveDrainTimeoutInMs;
```

Comment:
Why the need for a private variable, can we not use the constant directly?

Reply:
I made it a private variable so that I could change it for one of the live tests. This made it easier to make sure that the call to close the link happens before the service can respond that the link is drained.